examples/14_ampere_tf32_tensorop_gemm/ampere_tf32_tensorop_gemm_cute.cu

```diff
@@ -1,4 +1,5 @@
 /***************************************************************************************************
+ * Copyright (c) 2017 - 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Do we want both these lines here? (NVIDIA license line and Codeplay license line)

Reply:
Yes, the example is the same one we created for upstreaming. I just added the modifications to run it with SYCL.